pkg/controllers/management/catalog/controller.go

```diff
@@ -50,7 +92,17 @@ func Run(ctx context.Context, cacheRoot string, refreshInterval int, management
 	controller := management.Management.Catalogs("").Controller()
 	controller.AddHandler("catalog", m.Sync)
 
+	logrus.Infof("Starting project-level catalog controller")
+	projectCatalogController := management.Management.ProjectCatalogs("").Controller()
+	projectCatalogController.AddHandler("projectCatalog", m.PrjCatalogSync)
```

Comment:
Nit: do not use the exact object name for the handler; the name should reflect the functionality. Something like `projectCatalogSync` would be good